src/components/OSCALImplementedComponents.js

```diff
@@ -16,8 +16,12 @@ const ImplementedComponentsTable = styled(TableContainer)(
 );
 
 export default function OSCALImplementedComponents(props) {
-  const getComponent = (compUuid) =>
-    props.components?.find((component) => component.uuid === compUuid);
+  function getComponent(compUuid) {
+    if (Array.isArray(props.components)) {
+      return props.components?.find((component) => component.uuid === compUuid);
+    }
+    return props.components?.uuid === compUuid ? props.components : null;
```

Comment:
Wait can this actually happen? Have `components` that isn't an Array? Can you share a sample document where this happens and a link to the part of the OSCAL spec?

Reply:
So, I think some of our test data is out of date and a component was provided by itself and not in a list. I was thinking this check could be done in case the spec was ever changed, although this may be silly to include as it looks like this isn't supported currently.

Reply:
I believe `systemImplementationTestData` and `exampleSystemImplementation` in `SystemData` are causing the test failures. It looks like `components` doesn't contain a list.

Reply:
Okay yeah I think _our_ test data is bad here. Can you fix the test data?

Reply:
Test data is resolved now.